### PR TITLE
Update English notation.

### PR DESCRIPTION
**Pull request by ducdat0507**
So I:
- Make the second tier prefixes more accurate
- Fixed the small numbers sometimes have two dashes together instead of one
- Fixed the small numbers sometimes formatted as "x undefinedillion"

### CODE REVIEW

File: src/community/english.ts [hunk 1]
```diff
@@ -19,17 +18,17 @@ const PREFIXES = [
...
-  "hexaconti-", "mehexaconti-", "duehexaconti-", "trehexaconti-", "tetrehexaconti-", "pentehexaconti-", "hexehexaconti-", "heptehexaconti-", "octehexaconti-", "ennehexaconti-",
-  "heptaconti-", "meheptaconti-", "dueheptaconti-", "treheptaconti-", "tetreheptaconti-", "penteheptaconti-", "hexeheptaconti-", "hepteheptaconti-", "octeheptaconti-", "enneheptaconti-",
-  "octaconti-", "meoctaconti-", "dueoctaconti-", "treoctaconti-", "tetreoctaconti-", "penteoctaconti-", "hexeoctaconti-", "hepteoctaconti-", "octeoctaconti-", "enneoctaconti-",
-  "ennaconti-", "mecennaconti-", "duecennaconti-", "trecennaconti-", "tetrecennaconti-", "pentecennaconti-", "hexecennaconti-", "heptecennaconti-", "octecennaconti-", "ennecennaconti-",
-  "hecti-", "mehecti-", "duhecti-",
+  "", "milli-", "micro-", "nano-", "pico-", "femto-", "atto-", "zepto-", "yocto-", "xono-",
+  "veco-", "meco-", "dueco-", "treco-", "tetreco-", "penteco-", "hexeco-", "hepteco-", "octeco-", "enneco-",
+  "icoso-", "meicoso-", "dueicoso-", "trioicoso-", "tetreicoso-", "penteicoso-", "hexeicoso-", "hepteicoso-", "octeicoso-", "enneicoso-",
```
Comment (razenpok) [inline src/community/english.ts:23]: Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6279527)

**Comment by razenpok** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6279527&bid=20438024)

File: src/community/english.ts [hunk 2]
```diff
@@ -133,9 +131,8 @@ export class EnglishNotation extends EngineeringNotation {
 
   private formatPrefixes(e: number): string {
       e = Math.floor(e / 3) - 1;
-      // Quick returns
-      if (e == -1) return "";
-      if (e == 0) return "thousand";
+      // Quick returns.
+      if (e <= 3) return ["", "thousand", "million", "billion", "trillion"][e + 1];
```
Comment (razenpok) [inline src/community/english.ts:135]: Codacy found an issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6279527)

File: src/community/english.ts [hunk 3]
```diff
@@ -46,11 +45,11 @@ export class EnglishNotation extends EngineeringNotation {
   }
 
   public formatNegativeVerySmallDecimal(value: Decimal, places: number): string {
-    return `negative one ${this.formatDecimal(value.reciprocal(), places).replace(/ /g, '-')}th`;
+      return `negative one ${this.formatDecimal(value.reciprocal(), places).replace(/ /g, '-').replace('--', '-')}th`;
```
Comment (razenpok) [inline src/community/english.ts:48]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6279527)

File: src/community/english.ts [hunk 4]
```diff
@@ -46,11 +45,11 @@ export class EnglishNotation extends EngineeringNotation {
...
   }
 
   public formatVerySmallDecimal(value: Decimal, places: number): string {
-    return `one ${this.formatDecimal(value.reciprocal(), places).replace(/ /g, '-')}th`;
+      return `one ${this.formatDecimal(value.reciprocal(), places).replace(/ /g, '-').replace('--', '-')}th`;
```
Comment (razenpok) [inline src/community/english.ts:52]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6279527)

**Comment by dan-simon** [comment]
I merged the changes in this PR. It turned out that the double dashes were caused by double spaces which I also fixed.

**Pull request closed without merging**